src/server/data/index-data.json

```diff
@@ -1 +1,9 @@
-{"biography":"654","featuredPublications":[{"name":"3","description":"21","link":"32","type":"13"}],"currentProjects":[{"name":"21","description":"51","link":"5"}],"name":"321","age":"32"}
\ No newline at end of file
+{
```

Comment:
update file according to: https://github.com/teddy4445/lecture_website_template/blob/develop/data/jsons/index.json

Reply:
updated, please verify